### ISSUE TITLE
here is code run is not properly please fix the bug.it throw the error that .exe file is stop to working.

### ISSUE THREAD

**Issue by devensaini81** [microsoft/vscode-edge-devtools#1106]
**Environment (please complete the following information):**
 - OS (*e.g.,* Windows 10):
 - Extension version (*e.g.,* 1.2.3):

**Describe the bug:**

**Repro steps:**

**Expected behavior:**

**Additional context:**

**Comment by codepo8** [comment]
Yeah, that's unparseable and we don't have anything to do with any .exe
